rerun/src/main/java/org/apache/falcon/rerun/handler/RetryHandler.java

```diff
@@ -112,6 +112,8 @@ public void onSuccess(WorkflowExecutionContext context) throws FalconException {
     public void onFailure(WorkflowExecutionContext context) throws FalconException {
         // Re-run does not make sense when killed by user.
         if (context.isWorkflowKilledManually()) {
+            LOG.debug("User workflow: {} killed manually of instance: {} belongs to entity: {}, so retry won't happen ",
```

Comment:
Minor nit on the phrasing of the message. Can we rephrase to "Workflow {}; Instance {}; Entity {}, killed manually by user. Will not retry."